clover_admin.rb

```diff
@@ -220,6 +221,23 @@ def self.object_action(...)
       end
     end
 
+    model Vm do
+      order Sequel.desc(:created_at)
+      eager [:location, :vm_host]
+      columns [:name, :display_state, :vm_host, :location, :arch, :boot_image, :family, :vcpus, :created_at]
+      column_options display_state: {type: "select", options: ["running", "creating", "starting", "rebooting", "deleting"], add_blank: true},
+        arch: {type: "select", options: ["x64", "arm64"], add_blank: true},
+        family: {type: "select", options: Option::VmFamilies.map(&:name), add_blank: true},
+        vcpus: {type: "number"},
+        created_at: {type: "text"}
+
+      column_search_filter do |ds, column, value|
+        if column == :created_at
+          ds.where(Sequel.cast(:created_at, :text).ilike("%#{ds.escape_like(value)}%"))
```

Comment:
Since this pattern is used multiple times, I'd add a lambda and use it in each case to DRY things up:

```ruby
column_grep = lambda do|ds, column, value|
  ds.where(Sequel.cast(column, :text).ilike("%#{ds.escape_like(value)}%"))
end
```